contracts/entry/library.cairo

```diff
@@ -74,6 +91,14 @@ namespace Entries:
         return (sorted_entries_ptr)
     end
 
+    # @notice perform a bubblesort on an entries array
+    # @param num_entries: number of entries to use
+    # @param entries_ptr: pointer to first entry in array
+    # @param idx1: left index
+    # @param idx2: right index
```

Comment:
This should not be called except via the sort_entries_by_value function imo - would we still leave a comment?